core/schema/container.graphqls

```diff
@@ -111,6 +111,9 @@ type Container {
   """
   withoutMount(path: String!): Container!
 
+  "This container plus a socket forwarded to the given Unix socket path"
+  withUnixSocket(path: String!, source: SocketID!): Container!
```

Comment:
May as well add `withoutUnixSocket` now too I think

Reply:
Why ? 😇

Reply:
@grouville In general we always have some way to undo a modification to a container so that you can temporarily add a resource for setup commands then remove it for later ones.